spec/bolt_spec/run_spec.rb

```diff
@@ -123,35 +123,43 @@ def root_config
 
     describe 'apply_manifest' do
       it 'should apply a manifest file' do
+        bolt_inventory['features'] = ['puppet-agent']
```

Comment:
Sorry, I realize my line numbers were probably off after you deleted your variables! I meant for this to go in the `before(:all) do` and `after(:all) do` blocks.

Reply:
You can't use a `let` declaration in a `before(:all)` or `after(:all)` block, so there's no way to modify the `bolt_inventory` object used by the spec tests unless you do so inside the examples.